docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,75 @@ potentially untrusted network. This is why **we strongly recommend using
 a printer that does not have WiFi or Bluetooth**, and hooking up to it
 using a regular USB cable to print.
 
-Normally, any printer should work with Tails "out of the box." Most
-difficulties stem from not selecting the right driver (extra software
-needed for the printer and computer to communicate). Luckily, Tails has
-a large number of drivers for just about any popularly manufactured
-printer on hand, without even having to download new drivers from the
-web.
+Finding a printer that works with Tails can be challenging because Tails is
+based on the Linux operating system, which often has second-class hardware
+support in comparison to operating systems such as Windows or macOS.
+
+We `maintain a list of printers
+<https://docs.securedrop.org/en/stable/hardware.html#printers>`_ that we have
```

Comment:
When you are linking to other parts of the SecureDrop documentation, use [`:ref:`](http://www.sphinx-doc.org/en/stable/markup/inline.html#ref-role) or [`:doc:`](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-doc). Doing so is more semantically correct, and also preserves the ability to build all of the output targets supported by Sphinx without breakage.